tests/unit/test_eventstream.py

```diff
@@ -455,14 +457,19 @@ def test_event_stream_wrapper_iteration():
     parser.parse.assert_called_with(response_dict, output_shape)
 
 
-@raises(EventStreamError)
 def test_eventstream_wrapper_iteration_error():
-    raw_stream = create_mock_raw_stream(ERROR_EVENT_MESSAGE[0])
-    parser = Mock(spec=EventStreamXMLParser)
-    parser.parse.return_value = {}
-    output_shape = Mock()
-    event_stream = EventStream(raw_stream, output_shape, parser, '')
-    list(event_stream)
+    try:
```

Comment:
This should also done with a `with pytest.raises()` context manager.

Reply:
Ditto.